omod/src/main/webapp/resources/scripts/htmlForm.js

```diff
@@ -258,24 +278,50 @@
     // TODO: these methods (getEncounter*Date*) will have to be modified when/if we switch datepickers
     // TODO: could/should be generalized so as not to be datepicker dependent?
 
-    htmlForm.setEncounterStartDateRange = function(date) {
+    //Set encounterDate datepicker start date
+    htmlForm.setEncounterStartDateRange = function(date , handletimezones) {
         if (getField('encounterDate.value')) {
-            getField('encounterDate.value').datepicker('option', 'minDate', date);
+            if (handletimezones) {
+                var startDateClientTimezone = new Date(date)
```

Comment:
What does `startDateClientTimezone` mean? The start date or the client time zone?

Reply:
That is the start date with the client timezone. It transforms the start date with server TZ to client TZ

Reply:
In that case, i find the name confusing.

Reply:
What if i change it to `startDateWithClientTimezone` ?

Reply:
That would be excellent!